ArmyLibrary1/ArmyLibrary1/Control.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Army
+{
+    public class Control : Serviceman
+    {
+        public string CountyName;
+        public string Office;
+
+        public Control(string name, string surname, int number, string CountyName, string office) : base(name, surname, number)
+        {
+            CountyName = CountyName;
```

Comment:
Параметр конструктора лучше именовать с маленькой буквы. Тогда не будет такой путаницы
```suggestion
        public Control(string name, string surname, int number, string сountyName, string office) : base(name, surname, number)
        {
            CountyName = сountyName;
```